### DISCUSSION TITLE
May I know whether networkx support draw lines with/without direction in same graph?

### DISCUSSION THREAD

**Discussion by techgeniuswang** [networkx/networkx#7180] [Q&A] [answered]
Can anyone help confirm whether networkx support draw direct line and undirect line in same graph?

3 nodes in middle circle are same level, and they as leader in own organization. I want change lines in circle without direction

<img width="607" alt="image" src="https://github.com/networkx/networkx/assets/31273693/34a78002-420c-450e-8ba5-97869a3f3cff">

I tried ChatGPT, but still not find out how to implement it.

**Comment by dschult** [comment, marked as answer]
Draw the edges separately by specifying the `edgelist` input parameter to [nx.draw_networkx_edges](https://networkx.org/documentation/stable/reference/generated/networkx.drawing.nx_pylab.draw_networkx_edges.html) and calling it twice. Once with arrows turned on and then again with arrows turned off.
Of course, you will need to draw the nodes with the draw_networkx_nodes function too.
